test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -563,6 +563,140 @@ public void CanWriteToExcelAndUpdateData()
 
         }
 
+        [Test]
+        public void CanWriteJaggedArrayToExcel()
+        {
+            string openPath = Path.Combine(TestDirectory, @"core\excel\WriteJaggedArrayToExcel.dyn");
+            ViewModel.OpenCommand.Execute(openPath);
+
+            var filePath = System.IO.Path.GetTempPath() + Guid.NewGuid().ToString() + ".xlsx";
```

Comment:
This causes the temporary `*.xlsx` to accumulate over time and left there permanently. Try doing this instead:

```csharp
// Makes use of UnitTestBase.TempFolder since that folder is always 
// guaranteed to be different for each test, and will get deleted when 
// the test shuts down.
// 
var filePath = System.IO.Path.Combine(TempFolder, "output.xlsx");
```